pyproject.toml

```diff
@@ -0,0 +1,116 @@
+# build-system
+# - ref: https://peps.python.org/pep-0517/
+#
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+
+# project
+# - ref 1: https://peps.python.org/pep-0621/
+# - ref 2: https://hatch.pypa.io/latest/config/metadata/#project-metadata
+#
+[project]
+name = "jupyterhub-pytest-plugin"
+description = "A reusable JupyterHub pytest plugin"
+readme = "README.md"
+requires-python = ">=3.5"
+license = {file = "LICENSE"}
+keywords = ["jupyterhub", "pytest", "plugin"]
+authors = [
+    {name = "Jupyter Contributors", email = "jupyter@googlegroups.com"},
+]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python :: 3",
+]
+# TODO: List the packages and pin them to a version that your project depends on
+dependencies = [
+    # NOTE: If lower bounds are updated, also update our test for the lower
+    #       bounds in .github/workflows/test.yaml.
+    "jupyterhub>=2.0.0",
+]
+dynamic = ["version"]
+
+# TODO: List the packages and pin them to a version that the tests in your project depend on
+[project.optional-dependencies]
+test = [
+    "pytest>=5.4",
+    "pytest-cov",
+    "pytest-asyncio>=0.17",
+]
+
+[project.urls]
+Documentation = "https://jupyterhub-pytest-plugin.readthedocs.io"
```

Comment:
This doesn't yet exist. So let's open an issue about creating it and creating the skeleton `docs` directory also (ref https://jupyterhub-python-repo-template.readthedocs.io/en/latest/docs.html)

Reply:
Done